mkdocs-base.yml

```diff
@@ -145,7 +151,7 @@ plugins:
   with-pdf: # https://github.com/orzih/mkdocs-with-pdf
           output_path: '_pdf/PerconaServerforMongoDB-8.0.pdf'
           cover_title: 'Server for MongoDB Documentation 8.0'
-          cover_subtitle: 8.0.4-2 (February 19, 2025)
+          cover_subtitle: 8.0.6-3 (April 22, 2025)
```

Comment:
Release date was rescheduled to April 28

Reply:
Updated to 8.0.8-3